### DISCUSSION TITLE
Can I use this Vercel integration guide for Remix projects?

### DISCUSSION THREAD

**Discussion by offchan42** [getsentry/sentry-docs#9840] [Q&A]
This guide appears to be made for Next.js projects: https://docs.sentry.io/product/integrations/deployment/vercel/

Does it still work if I use Remix project instead?
I need a way for Vercel to notify Sentry on every deployment in a Remix project and upload sourcemaps.

**Comment by lforst** [comment]
(quietly gonna answer this one year late) We'll take a look. The integration just adds a few env vars to your project and in theory you can pick them up from within remix/RR7.
